### ISSUE TITLE
Huge memory leaks, desperate

### ISSUE THREAD

**Issue by BenDes21** [lllyasviel/stable-diffusion-webui-forge#2391]
Hello there, since few days and after an update of my Nvidia drivers + a power outage, my laptop ( asus strix 4080rtx ) seems have some serious problem with the Vram, it's impossible to generate an image with Forge without getting a huge memory leaks.

I already reinstalled an older version of my nvidia drivers, clean the fan's of my laptop and even completely reset my laptop ( reinstall a clean version of windows 11 with only Forge on it ) nothing seems to fix the problem who appeared suddenly ( before all was running perfectly )

Would like to know if the issue come from directly my laptop ( during the power outage, my laptop was plugged into a power stripe with the battery full so Im not sure ) or if it's simply forge with the last commit with seems to not working with my specific config.

If anyone has the solution it's will help me a lot :( Thanks

![forge-memory-leaks-desperate-v0-6wg0tusvnl3e1](https://github.com/user-attachments/assets/109b1fd8-df1f-4502-b8a9-ca6aa73e6b5f)

**Comment by Volnovik** [comment]
I can confirm this. Was not using forge for couple of weeks and now it struggles with full model on 4090. 
Seems like offloading vram is off. It leaves something in shared (more than previously) and that part is increasing. Also al some point I got 63 of 64 GB of ram used which is also abnormal. But on the other hand it does not happen every time.

**Comment by andy8992** [comment]
had to restart forge just now, it was using 78gb of my pagefile lol

**Comment by rockiecxh** [comment]
indeed, I met similar issues. restart the forge can fix the issue, just as a workaround

**Comment by rafalswierczek** [comment]
I confirm problems with RAM: #2401

**Comment by Volnovik** [comment]
> indeed, I met similar issues. restart the forge can fix the issue, just as a workaround

Yes, but previously it was once in few days. Now it is once per ~5 generations.

**Comment by rockiecxh** [comment]
today I use lora to gen images, even though I checked never OOM, but it still OOM after running some gen.

**Comment by Vtamins** [comment]
As a running engineer in a Kubernetes pod for generate image, I encounter the same issue with Forge: it restarts every day due to exceeding the memory limit

**Comment by Vtamins** [comment]
The same issue occurs with Forge running as an engine in a Kubernetes pod: it restarts every day because it exceeds the memory limit

**Comment by alexdisablo** [comment]
Same issue here

**Comment by Vtamins** [comment]
您的邮件已经收到！

**Comment by MrJackSpade** [comment]
Updated from an older version to use Flux and now I'm seeing this as well. I can gen 2-3 images before it crashes

**Comment by Vtamins** [comment]
您的邮件已经收到！

**Comment by pepepaco** [comment]
i had to revert and use the old version that use torch 2.1, 
[Forge with CUDA 12.1 + Pytorch 2.1](https://github.com/lllyasviel/stable-diffusion-webui-forge/releases/download/latest/webui_forge_cu121_torch21.7z)
